gnome-image-installer/pages/install/gis-scribe.c

```diff
@@ -1072,7 +1072,7 @@ gis_scribe_begin_verify (GisScribe *self,
     g_pollable_input_stream_create_source (G_POLLABLE_INPUT_STREAM (gpg_stdout),
                                            cancellable);
   g_task_attach_source (task, task_data->stdout_source,
-                        (GSourceFunc) gis_scribe_gpg_progress);
+                        (GSourceFunc) (void (*)(void)) gis_scribe_gpg_progress);
```

Comment:
I think upstream would probably be interested in a `G_SOURCE_FUNC()` casting macro for this. But to avoid dependency faff, you should go forth with `(void (*)(void))` here for now.

Reply:
*hopeful smile*

Reply:
MVP at https://gitlab.gnome.org/GNOME/glib/merge_requests/82